src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -176,6 +201,8 @@ class boss_bronjahm : public CreatureScript
 
                 DoMeleeAttackIfReady();
             }
+        private:
+            uint32 _soulFragmentsSpawned;
```

Comment:
no. initialize the variable _soulFragmentsSpawned = 0; withing the brackets and thats it :)